### PR TITLE
chor(docs): Improved Optimize Public API Documentation

### PR DESCRIPTION
**Pull request by grlimacan**
## What is the purpose of the change

Improving the documentation on how to get a valid token for C8 SaaS mode

## Are there related marketing activities

No

## When should this change go live?

The feature is already existent, this is an improvement of the docs, so immediately

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: optimize/apis-clients/optimize-api/optimize-api-authorization.md [hunk 1]
```diff
@@ -15,3 +15,46 @@ Authorization: Bearer mySecret
 The token used to access the Optimize API can be a configurable shared secret (except in Camunda Platform 8 SaaS mode) or a JWT compliant with the OAuth2 Protocol (all modes).
 
 Refer to [Public API Configuration](../../self-managed/optimize-deployment/configuration/system-configuration.md#public-api) for the particular configuration to access the public API using a token.
+
+### How to obtain the access token for C8 SaaS (Cloud) usage
+
+You must obtain a token to use the Optimize API. When you create a Optimize [client](/guides/setup-client-connection-credentials.md), you get all the information needed to connect to Optimize.
```
Comment (andromaqui) [inline optimize/apis-clients/optimize-api/optimize-api-authorization.md:21]: have we tested that the link works?

File: optimize_versioned_docs/version-3.8.0/apis-clients/optimize-api/optimize-api-authorization.md [hunk 2]
```diff
@@ -15,3 +15,46 @@ Authorization: Bearer mySecret
 The token used to access the Optimize API can be a configurable shared secret (except in Camunda Platform 8 SaaS mode) or a JWT compliant with the OAuth2 Protocol (all modes).
 
 Refer to [Public API Configuration](../../self-managed/optimize-deployment/configuration/system-configuration.md#public-api) for the particular configuration to access the public API using a token.
+
+### How to obtain the access token for C8 SaaS (Cloud) usage
+
+You must obtain a token to use the Optimize API. When you create a Optimize [client](/guides/setup-client-connection-credentials.md), you get all the information needed to connect to Optimize.
```
Comment (andromaqui) [inline optimize_versioned_docs/version-3.8.0/apis-clients/optimize-api/optimize-api-authorization.md:21]: It should be: "When you create an Optimize client"

same in other files also

**Review by RomanJRW — APPROVED** [review]
LGTM generally, just one tiny grammar suggestion

**Review by andromaqui — APPROVED** [review]
Just a couple of remarks :)

File: optimize/apis-clients/optimize-api/optimize-api-authorization.md (hunk 1 above)
Comment (grlimacan) [reply optimize/apis-clients/optimize-api/optimize-api-authorization.md:21]: Seems to not be working, I'm correcting

**Review by andromaqui — APPROVED** [review]
LGTM ☄️

**Pull request merged**